app/database/connection.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+
+class Connection():
+    _Session = None
+
+    @classmethod
+    def get(cls, url):
+        if cls._Session is None:
+            engine = create_engine(url, echo=False)
+            cls._Session = sessionmaker(bind=engine)
+        return cls._Session()
```

Comment:
Actually after thinking about it more, I don't think we should disallow the ability to create new sessions. I do like and support the idea of separating the `get` into an `init` or `create` for actually creating the session, but we should be able to create a new session if we explicitly want to